mantidproject/mantid
# Add SNS instruments to PyChop

**Description of work.**

**Report to:** A. Savici / G. Sala (SNS)

**To test:**


* Build and run. Interface->Direct->PyChop. 
* Check that the SNS instruments (`ARCS`, `CNCS`, `HYSPEC` and `SEQUOIA`) are now included in the instruments list
* Select `HYSPEC` - check there is an extra edit box labelled `S2`. Click `Calculate and Plot`. Click on the `Q-E` tab. Now change the `S2` value and click `Calculate and Plot` again, verify that the `Q-E` tab plot changes. The wedge should move to a lower |Q| if `S2` is low, and vice versa.
* Select each of the SNS instruments and check that the frequencies are multiples of 60Hz. (ISIS instruments would have multiples of 50Hz for TS1 or 10Hz for TS2).

Fixes #29966






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
